Controllers/MenuViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  MenuViewController.swift
+//  Clic&Farm-iOS
+//
+//  Created by Guillaume Roux on 03/12/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class MenuViewController: UITableViewController {
+
+  var commands: [Command]!
+
+  // MARK: - Initialization
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    initializeCommands()
+    setupNavigationBar()
+    setupTableView()
+  }
+
+  private func initializeCommands() {
+    let termsOfUseCommand = Command(name: "terms_of_use", assetName: "doc", closure: presentTermsOfUse)
+    let logoutCommand = Command(name: "logout", assetName: "logout", closure: presentLogoutAlert)
+
+    commands = [termsOfUseCommand, logoutCommand]
+  }
+
+  private func setupNavigationBar() {
+    let menuImageView: UIImageView = {
+      let menuImageView = UIImageView(frame: CGRect.zero)
+      menuImageView.contentMode = .scaleAspectFit
+      menuImageView.image = UIImage(named: "menu")?.withRenderingMode(.alwaysTemplate)
+      menuImageView.tintColor = UIColor.darkGray
+      menuImageView.translatesAutoresizingMaskIntoConstraints = false
+      return menuImageView
+    }()
+
+    navigationItem.titleView = menuImageView
+  }
+
+  private func setupTableView() {
+    tableView.register(CommandCell.self, forCellReuseIdentifier: "CommandCell")
+    tableView.rowHeight = 50
+    tableView.tableFooterView = UIView()
+    tableView.delegate = self
+    tableView.dataSource = self
+  }
+
+  // MARK: - Table view
+
+  override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
What's the use of this?
When will it need to return something else than 1?

Reply:
We don't need something else than 1 here, so this is useless to override this method. I just removed it.